rest/taskrouter/statistics/taskqueue/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,26 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using Newtonsoft.Json.Linq;
+using System;
+using Twilio;
+using Twilio.Rest.Taskrouter.V1.Workspace.TaskQueue;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        const string workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string taskQueueSid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        TwilioClient.Init(accountSid, authToken);
+
+        var stats = TaskQueueStatisticsResource.Fetch(
+                        workspaceSid, taskQueueSid, DateTime.Now, 500);
+
+        var cumulativeStats = JObject.FromObject(stats.Cumulative);
+
+        Console.WriteLine(cumulativeStats["reservations_accepted"].Value<int>());
```

Comment:
There is no need to cast the value to `int`. `Console.Writeline` will cast this value as an string to display it in show this in the console.

Please consider using:

```csharp
Console.WriteLine(cumulativeStats["reservations_accepted"]);
```